graphbolt/src/index_select.cc

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file index_select.cc
+ * @brief Index select operators.
+ */
+#include "./index_select.h"
+
+#include "./macro.h"
+
+namespace graphbolt {
+namespace ops {
+
+torch::Tensor UVAIndexSelect(torch::Tensor input, torch::Tensor index) {
```

Comment:
Do we want a UVAIndexSelect api or just IndexSelect and dispatch to different code by check the status of input tensor and index tensor?

Reply:
A UVAIndexSelect is sufficient because we will dispatch other cases to Pytorch kernel.